### ISSUE TITLE
docs website doesn't work in IE11

### ISSUE THREAD

**Issue by malreynolds** [zippyui/react-date-picker#129]
I have a flex issue with the datepicker on IE11. Wanted to test if it's there's something local that affects it so I went onto http://zippyui.com/docs/react-date-picker and the page doesn't load at all.

Testing through browserstack, Windows 10 IE11

**Comment by dev-zippyui** [comment]
Yes, we're aware of the issues with IE 11 on our website, and will solve those as soon as our datepicker docs are complete. Thanks for reporting!
